schema_based_on_diagram.sql

```diff
@@ -1 +1,51 @@
+CREATE TABLE patients (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(255),
+  date_of_birth DATE,
+  PRIMARY KEY(id)
+);
 
+CREATE TABLE medical_histories (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  admitted_at TIMESTAMP,
+  patient_id INT REFERENCES patients(id),
+  status VARCHAR(255),
+  PRIMARY KEY(id)
+);
+
+CREATE INDEX patient_histories ON medical_histories(patient_id);
+
+CREATE TABLE treatment_history (
+  history_id INT REFERENCES medical_histories(id),
+  treatment_id INT REFERENCES treatments(id)
```

Comment:
- [ ] Not all FOREIGN KEYS have indexes, kindly create index for `history_id` and `treatment_id` in treatment_history
e.g
```sql
CREATE INDEX ON treatment_history (history_id);
CREATE INDEX ON treatment_history (treatment_id);
```